stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -316,6 +316,41 @@ extension Unsafe${Mutable}RawBufferPointer: ${Mutable}Collection {
     }
     return 0
   }
+
+  public func _customIndexOfEquatableElement(_ element: UInt8) -> Index?? {
+    guard !isEmpty else { return Optional(nil) }
+
+    let ptr = baseAddress._unsafelyUnwrappedUnchecked
```

Comment:
There is no reason to unsafely unwrap here.
```suggestion
    guard let ptr = baseAddress else { return Optional(nil) }
```

Reply:
fixed